corehq/apps/translations/app_translations/download.py

```diff
@@ -381,8 +381,10 @@ def get_form_question_label_name_media(langs, form):
                 itext_items[text_id][(lang, value_form)] = value
 
     itext_items['submit_label'] = {}
+    itext_items['submit_notification_label'] = {}
     for lang in langs:
         itext_items['submit_label'][(lang, 'default')] = form.get_submit_label(lang)
+        itext_items['submit_notification_label'][(lang, 'default')] = form.get_submit_notification_label(lang)
```

Comment:
we spoke about this offline, but I think it'd make sense to move these two to a separate helper function, as this one deals with form questions.  That's also allow for dropping the "allow empty for submit_notification_label row" stuff.